openvdb_houdini/houdini/SOP_OpenVDB_Convert.cc

```diff
@@ -427,8 +427,11 @@ Polygon Soup:\n\
     obsoleteParms.add(hutil::ParmFactory(PRM_INT_J, "activepart", ""));
 
     // Register this operator.
-    hvdb::OpenVDBOpFactory("OpenVDB Convert",
+    hvdb::OpenVDBOpFactory("Convert VDB",
```

Comment:
I don't suppose we could leave this as "VDB Convert", so that it sorts with all the other SOPs?

Reply:
I think I'd prefer that we align ourselves with Houdini's names as much as possible. Most Houdini users are more familiar with "Convert VDB", so we're breaking with their expectation by naming it "VDB Convert" instead. 

As I see it working, when we add a new SOP to the library, @jmlait decides if it's a good fit for Houdini and then chooses whether to change the label name or not. If we disagree with a rename, we can lobby Jeff to try and change his mind, but ultimately if he decides to rename, we should adopt the new name as the open-source name as well. Ideally this whole process should happen before the PR with the new SOP gets merged so that no-one even gets to see the original proposed name.

There is of course some legacy to resolve, but unless there's a very good reason to do so, I think the renames that have already been decided on by SideFX are the names we should migrate to.

Reply:
What about VDB Create, then?  Should that be renamed to "VDB"?

Reply:
No, unfortunately there's not a 1:1 in functionality between the VDB SOP and the VDB Create SOP.

The VDB SOP allows you to build a frustum grid without needing a camera, manually specifying taper, depth, etc. That's pretty useful. The VDB Create SOP doesn't offer this.

The VDB Create SOP has a "match topology" option which activates topology matching a reference VDB on creation. This workflow is possible using native VDB SOPs, but it's more fiddly. You have to follow the VDB SOP with a VDB Activate SOP, switch to the reference tab, enable the "activate using VDB" option and then deselect the value toggle. 

Nonetheless, I believe most users are more comfortable with the native SOP workflow, so if SideFX are willing to donate them, I think we should adopt the SideFX SOPs here (VDB SOP and VDB Activate SOP) and then deprecate (ie ophide) the VDB Create SOP rather than simply renaming it.

Reply:
>   No, unfortunately there's not a 1:1 in functionality between the VDB SOP and the VDB Create SOP.

Likewise for Convert VDB and Convert VDB (ASWF), though.

Reply:
> Likewise for Convert VDB and Convert VDB (ASWF), though.

What are the differences in functionality between these two? I've just been through the options on these two SOPs and the parameters, node inputs and default values look identical to me? (testing on 17.0.456).

Reply:
Through 16.5, the value type conversion options are missing.  If they've been added in 17, then the docs are out of date.

Reply:
Yeah, it seems like those parameters have been added to the Convert VDB SOP in H17 ("vdbtype", "vdbprecision"), but the Houdini help card hasn't been updated (@jmlait).

Reply:
Just my thoughts on this. I was going to suggest changing the Convert VDB to VDB Convert mainly because it would be nice to keep them all the same and have them sorted alphabetically in the node list in houdini. The way it is now, they appear at the bottom of the list when you type VDB as the filter.

Reply:
@dokipen3d - I don't disagree with you. However, SideFX has obviously renamed it so that the "Convert VDB" SOP fits in more consistently with the "Convert" SOP. Unless there's a very good reason and I consider your reasoning here as being more of a nice-to-have, I think we should align ourselves with Houdini naming. 

Two immediate benefits of this - an artist that moves from company X where the open-source nodes are not installed to company Y where they are won't have to think twice about where to find this SOP. Also a company where the open-source nodes are suddenly switched on where they weren't before will have less of a jarring experience in making that transition. :)

Reply:
@danrbailey yeah good points. One option could be for sidefx to wrap up the convert functionality into the Convert SOP and deprecate Convert VDB. Old scenes could still open but newly created nodes could be the possible VDB Convert. I don't think this should be done now but something to think about for the future. I've just been over the shoulders of many artists who try to drop down a convert and there's always a pause and a mental jump to the fact that its not VDB _X._

Reply:
Convert SOP is a giant kitchen sink, so adding the various VDB specific choices to it is a bit of a nightmare, which is why we were happy with a separate Convert VDB.   The word-order doesn't matter for tab-typing as you can type either way, but it is true it shows up at the bottom.   We picked that one as we already have a class of "Convert Foo" nodes.

Good point about the missing docs on vdbtype and vdbprecision!  That was an oversight on my part.  Unfortunately I have to manually move the parameter-help into a separate .txt help card (as we have to have a single .txt help card for node help as we don't want to force technical writers to edit C++ files!)  Bug: 95181 for the missing help.